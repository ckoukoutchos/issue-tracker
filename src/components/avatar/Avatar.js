import React from 'react';
import styles from './Avatar.module.css';

const Avatar = ({ url }) => {
  return <img className={styles.avatar} src={url} alt="user avatar" />
}

export default Avatar;