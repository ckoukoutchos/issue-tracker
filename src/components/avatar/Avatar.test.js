import React from 'react';
import Avatar from './Avatar';
import renderer from 'react-test-renderer';

it('renders the Avatar component correctly', () => {
  const tree = renderer
    .create(<Avatar url="http://madupsite.com" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});