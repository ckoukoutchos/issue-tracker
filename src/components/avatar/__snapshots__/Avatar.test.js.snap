// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Avatar component correctly 1`] = `
<img
  alt="user avatar"
  className="avatar"
  src="http://madupsite.com"
/>
`;

exports[`renders the Card component correctly 1`] = `
<img
  alt="user avatar"
  className="avatar"
  src="http://madupsite.com"
/>
`;
